test/gurka/test_instructions_obvious_maneuver.cc

```diff
@@ -0,0 +1,128 @@
+#include "gurka.h"
```

Comment:
Thank you for these tests! Its really easy to visualize how the continue collapsing is working! 🙏